constants.js

```diff
@@ -84,5 +103,11 @@ module.exports = {
     VOLUNTEER: VOLUNTEER,
     STAFF: STAFF,
     GODSTAFF: GODSTAFF,
+    SPONSOR_T1: SPONSOR_T1,
+    SPONSOR_T2: SPONSOR_T2,
+    SPONSOR_T3: SPONSOR_T3,
+    SPONSOR_T4: SPONSOR_T4,
+    SPONSOR_T5: SPONSOR_T5,
```

Comment:
Do we actually have 5 different tiers of sponsors?

Reply:
Yeah I saw some notion/there are 5 tiers in the Roles